# This message holds state/command data of following joints: 
#   - polar_joint1
#   - polar_joint2
#   - polar_joint3
#   - polar_joint4
#   - polar_joint5
#   - polar_joint6
#   - polar_hand_joint1

float64[7] position # rad
float64[7] velocity # rad/s